k2/python/csrc/torch/fsa_algo.cu

```diff
@@ -99,15 +99,17 @@ static void PybindLinearFsa(py::module &m) {
 static void PybindIntersect(py::module &m) {
   m.def(
       "intersect",  // works only on CPU
-      [](FsaOrVec &a_fsas, FsaOrVec &b_fsas,
+      [](FsaOrVec &a_fsas, int32_t properties_a,
```

Comment:
Please also update line 128 in this file to add arguments for `properties_a` and `properties_b`. I guess the current code won't compile.

Reply:
sorry I forgot to push my local changes to that